components/formats-bsd/src/loci/formats/in/FakeReader.java

```diff
@@ -467,6 +467,13 @@ protected void initFile(String id) throws FormatException, IOException {
         }
       }
 
+      table = list.getTable(list.getHeaders().get(0));
```

Comment:
Definitely! Good catch. This was debugging code since my previous use of `RAF.writeChars` led to the getTable method not working. Pushing a fix now.